test/services/actionService.test.js

```diff
@@ -0,0 +1,18 @@
+
+const actionsService = require('../../server/services/actionsService')
+describe('actionsService', () => {
+  test('actions service returns a actions object containing an array', async () => {
```

Comment:
This description doesn't quite match the assertion below which expects just an array, not an object containing an array.

Reply:
Fixed